srcpkgs/elog/files/elogd/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec chpst -u elog:elog /usr/bin/elogd -c /etc/elog/elogd.cfg
```

Comment:
Actually, why can't you just give it `/etc/elogd.cfg` unless this is where upstream thinks the conf file should go?

Reply:
upstream recommends a subdirectory in which SSL keys can be placed, too.
source: https://elog.psi.ch/elog/adminguide.html